adaptdl/adaptdl/torch/scaling_rules.py

```diff
@@ -156,14 +129,14 @@ def var_avg(self):
 
         Returns (float): Estimate of trace of the covariance.
         """
-        return float(np.sum(np.maximum(self._state["var_avg"], 1e-6)))
+        return float(np.sum(np.maximum(self.state["var_avg"], 1e-6)))
 
     def get_progress(self):
-        return self._state["progress"]
+        return self.state["progress"]
```

Comment:
Implement `@property` methods for instead of exposing the entire `state` as a public field.